sdks/python/apache_beam/runners/direct/direct_runner.py

```diff
@@ -90,15 +92,14 @@ class DirectRunner(PipelineRunner):
   """Executes a single pipeline on the local machine."""
 
   # A list of PTransformOverride objects to be applied before running a pipeline
-  # using DirectRunner.
-  # Currently this only works for overrides where the input and output types do
-  # not change.
-  # For internal SDK use only. This should not be updated by Beam pipeline
-  # authors.
+  # using DirectRunner. Currently, this only works for overrides where the input
+  # and output types do not change.
+  # For internal use only; no backwards-compatibility guarantees.
   _PTRANSFORM_OVERRIDES = []
 
   def __init__(self):
     self._cache = None
+    self._use_test_clock = False  # use RealClock() in production
```

Comment:
<!--thread_id:cc_153932413_t; commit:517fae9dcb749a59322ea40c7de0eb90ebf5efac; resolved:1-->
<!--section:context-quote-->
> **charlesccychen** wrote:
> Can you document that this is used for testing when TestStream is used in the pipeline?

<!--section:body-->
I mention that a few lines below, when I turn on the flag.